src/AppBundle/Manager/Project/BugManager.php

```diff
@@ -27,22 +29,27 @@ class BugManager
     protected $userManager;
     /** @var Parser **/
     protected $parser;
-    
+    /** @var UrlGeneratorInterface **/
+    protected $router;
+
     /**
      * @param FeedbackGateway $gateway
      * @param CommentManager $commentManager
      * @param LabelManager $labelManager
      * @param NotificationManager $notificationManager
      * @param UserManager $userManager
      * @param Parser $parser
+     * @param Router $router
```

Comment:
This line must be removed

Reply:
Or rather modified to @param UrlGeneratorInterface. I made the error, then copied and pasted it. I forgot to correct everywhere.

Reply:
Yeah, obviously.